plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestBaseJdbcConfig.java

```diff
@@ -50,9 +53,9 @@ public void testExplicitPropertyMappings()
         Map<String, String> properties = ImmutableMap.<String, String>builder()
                 .put("connection-url", "jdbc:h2:mem:config")
                 .put("jdbc-types-mapped-to-varchar", "mytype,struct_type1")
-                .put("metadata.cache-ttl", "1s")
+                .put(METADATA_CACHE_TTL, "1s")
                 .put("metadata.cache-missing", "true")
-                .put("metadata.cache-maximum-size", "5000")
+                .put(METADATA_CACHE_MAXIMUM_SIZE, "5000")
```

Comment:
Here constants shouldn't be used.
This test covers parsing of configuration, so it should use literals to prevent inadvertent changes

Reply:
Reverted.